Makefile

```diff
@@ -70,10 +70,12 @@ endif
 ifeq (rocksdb,$(findstring rocksdb,$(COSMOS_BUILD_OPTIONS)))
   CGO_ENABLED=1
```

Comment:
is this meant to be left around for users who want to use it in tm?

Reply:
yeah - but it's more complicated than that (I think we discussed this in our chat)